src/index.html

```diff
@@ -1,13 +1,48 @@
 <!doctype html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="stylesheet" href="./style.css">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <div class="product-card__image">
+        <img src="./images/imac.jpeg" alt="iMac">
+      </div>
+      <p class="product-card__title">
+        Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+      </p>
+      <p class="product-card__product-code">
+        Код товара: 195434
+      </p>
+      <div class="response">
+        <div class="response__stars">
+          <div class="response__star"></div>
+          <div class="response__star"></div>
+          <div class="response__star"></div>
+          <div class="response__star"></div>
+          <div class="response__star"></div>
+        </div>
+        <p class="response__count">
+          Отзывов: 5
+        </p>
+      </div>
+      <div class="product-card__price">
+        <p class="product-card__price-text">
```

Comment:
гадаю що це має бути також заголовок
```suggestion
        <h3 class="product-card__price-title">
```